src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -306,19 +306,21 @@ public static SyntaxTree ParseWithRoundTripCheck(string text, CSharpParseOptions
             string ilSource,
             IEnumerable<MetadataReference> references = null,
             CSharpCompilationOptions options = null,
-            bool appendDefaultHeader = true) => CreateCompilationWithILAndMscorlib40(source, ilSource, TargetFramework.Standard, references, options, appendDefaultHeader);
+            CSharpParseOptions parseOptions = null,
+            bool appendDefaultHeader = true) => CreateCompilationWithILAndMscorlib40(source, ilSource, TargetFramework.Standard, references, options, parseOptions, appendDefaultHeader);
```

Comment:
>bool appendDefaultHeader = true) => CreateCompilationWithILAndMscorlib40(source, ilSource, TargetFramework.Standard, references, options, parseOptions, appendDefaultHeader); [](start = 12, length = 173)

Given a number of parameters for the method and the presence of optional ones, the expression body form is "unreadable". Consider converting this method to the regular body form.